pages/dkp/konvoy/2.1/supported-operating-systems/index.md

```diff
@@ -15,7 +15,8 @@ Konvoy supports the following base Operating Systems.
 ## Amazon Web Services (AWS)
 
 <!-- vale Vale.Spelling = NO -->
-| Operating System      | Kernel                      | Default Config | FIPS | Air Gapped | FIPS with Air Gapped | GPU Support <!-- vale Vale.Spelling = YES --> |
+<!-- vale Vale.Terms = NO -->
+| Operating System      | Kernel                      | Default Config | FIPS | Air Gapped | FIPS with Air Gapped | GPU Support <!-- vale Vale.Spelling = YES --> <!-- vale Vale.Terms = YES --> |
```

Comment:
this seems highly unusual. You said the comments won't work below the table?

Reply:
I just wanted it to take affect for this one line is all.

Reply:
I think to keep it very limited in scope, in the table is good.

Reply:
So, yeah - @vacas5 - the vale linter is happy again now.

Reply:
I just want to make sure it doesn't render in the preview